### PR TITLE
Task common layout

### PR DESCRIPTION
**Pull request by obwansan**
Created common layout / structure of site pages.

### CODE REVIEW

File: home.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    background: #000046;  /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #000046, #1CB5E0) fixed;  /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #000046, #1CB5E0) fixed; /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+    /*Have to put 'fixed' on end of linear-gradient for it to work / fill entire screen*/
+}
+
+.name {
+    text-align: center;
+    font-size: 3rem;
+    /*border: 1px solid #FFF;*/
```
Comment (C0deJack) [inline home.css:12]: might be worth deleting unused code

File: index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Portfolio Site</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" type="text/css" href="home.css">
+    <link href="https://fonts.googleapis.com/css?family=Josefin+Sans" rel="stylesheet">
```
Comment (C0deJack) [inline index.html]: normalise.css could be useful

File: index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+</head>
+<body>
+    <main>
+        <div class="outer-container">
+            <div class="inner-container">
+                <div class="name">Kevin O'Brien</div>
+                <div class="title">Web Developer</div>
+            </div>
+            <div class="page-links-container-home">
```
Comment (C0deJack) [inline index.html:17]: class names are quite long

File: portfolio.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    background-size: contain;
+    /* Vertically and horizontally centre div */
+    position: relative;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.page-links-container-portfolio {
+    /* Vertically and horizontally centre div */
```
Comment (C0deJack) [inline portfolio.css:33]: these comments should be self explanatory but if its useful to you..

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    font-family: 'Josefin Sans', sans-serif;
+    color: #FFF;
```
Comment (C0deJack) [inline styles.css]: maybe keep hex codes all 6 digit

File: home.css (hunk 1 above)
Comment (obwansan) [reply home.css:12]: will do

File: index.html (hunk 2 above)
Comment (obwansan) [reply index.html]: will do

File: index.html (hunk 3 above)
Comment (obwansan) [reply index.html:17]: true...

File: portfolio.css (hunk 4 above)
Comment (obwansan) [reply portfolio.css:33]: Yep. Going to save a version with comments for future reference.

File: styles.css (hunk 5 above)
Comment (obwansan) [reply styles.css]: yep

File: home.css (hunk 1 above)
Comment (obwansan) [reply home.css:12]: ...but keeping in for now as am using it to check layout

**Comment by C0deJack** [comment]
All good - passed second code review

**Pull request merged**